### PR TITLE
Added eslint-plugin-flowtype-errors to readme

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,15 +65,15 @@ can now utilize the `babel-eslint` parser to pass in sanatized JavaScript code.
...
+The `eslint` plugin `eslint-plugin-flowtype` will mute those warnings, so let
 us install all the eslint stuff we need:
 
 ```
 # ESlint stuff (you probably already did that)
 npm install eslint babel-eslint --save-dev
 
 # Now the flow related stuff
-npm install eslint-plugin-flow-vars
+npm install eslint-plugin-flowtype
```
Comment (ryyppy) [inline README.md:76]: We just used the plugin and we found that doing the recommended settings works best:
https://github.com/gajus/eslint-plugin-flowtype#recommended

Should we mention that?

**Review by ryyppy — APPROVED** [review]
Okay I will add details myself ... thanks for the contribution :-)

File: README.md (hunk 1 above)
Comment (amilajack) [reply README.md:76]: Ideally, this should work out of the box. I'll work with the `eslint-plugin-flowtype` collaborators to make this work with no configuration if its not already.

Comment (amilajack) [reply README.md:76]: Seems like this would work:

```
{
  "extends": [
    "plugin:flowtype/recommended"
  ],
  "plugins": [
    "flowtype"
  ]
}
```

Comment (ryyppy) [reply README.md:76]: Yeah, we used something like this... either we should refer to the plugin or add the config explicitly, I guess

**Pull request merged**